gestore/processors.py

```diff
@@ -0,0 +1,89 @@
+from typing import Any, List, Optional, Tuple, Union
+
+from django.core.exceptions import ObjectDoesNotExist
+from django.db.models import (
+    ForeignKey,
+    ManyToManyField,
+    ManyToManyRel,
+    ManyToOneRel,
+    Model,
+    OneToOneField,
+)
+
+
+def process_foreign_key(
+        instance: Model,
+        field: ForeignKey
+) -> Tuple[Any, Model]:
+    """
+    What we are looking to achieve here is to get the ID of the object this
+    instance is pointing at, and to return that instance for later processing.
+
+    Note: This will process both; ForeignKeys and OneToOneKey. As in
+    Django a OneToOneKey is sub class of ForeignKey.
+    """
+    # Gets the ID of the instance pointed at
+    value = field.value_from_object(instance)
+    return value, getattr(instance, field.name)
+
+
+def process_one_to_many_relation(
+        instance: Model,
+        field: ManyToOneRel
+) -> List[Model]:
```

Comment:
Is this Python 3.5 syntax? It seems odd to me.

Reply:
It is. These function are a little bit odd because no one had to use them before, I'm just adding a little hint on what's supplied, and what to expect as an output from each function so that we reduce confusion.